src/test/java/jenkins/branch/buildstrategies/basic/AllBranchBuildStrategyImplTest.java

```diff
@@ -100,13 +96,7 @@ public void given__false_strategies__when__isAutomaticBuild__then__returns_false
                 new AllBranchBuildStrategyImpl(Arrays.asList(
                     new BranchBuildStrategy() {
                         @Override
-                        public boolean isAutomaticBuild(@NonNull SCMSource scmSource, @NonNull SCMHead scmHead, @NonNull SCMRevision scmRevision, SCMRevision scmRevision1, TaskListener taskListener) {
-                            return false;
-                        }
-                    },
-                    new BranchBuildStrategy() {
-                        @Override
-                        public boolean isAutomaticBuild(@NonNull SCMSource scmSource, @NonNull SCMHead scmHead, @NonNull SCMRevision scmRevision, SCMRevision scmRevision1, TaskListener taskListener) {
+                        public boolean isAutomaticBuild(@NonNull SCMSource source, @NonNull SCMHead head, @NonNull SCMRevision currRevision, SCMRevision lastBuiltRevision, SCMRevision lastSeenRevision, TaskListener taskListener) {
```

Comment:
Given the description of the test cases _(strategies not strategy)_ The extra added `BranchBuildStrategy` should not have been removed. The same seems to go for all the tests where you have removed them.